CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,73 @@
+# Changelog 2.1
+
+## 2.1.0 - 2024-07-26
+
+### Added
+
+- Add a new feature `BGPPolicy` to allow users to initialize BGP process on selected Kubernetes Nodes and advertise Service IPs, Pod IPs, and Egress IPs to remote BGP peers.
+  - Add new `BGPPolicy` API and CRD Controller. ([#6009](https://github.com/antrea-io/antrea/pull/6009) [#6203](https://github.com/antrea-io/antrea/pull/6203), [@hongliangl])
+  - Add BGP datapath interface and implement goBGP integration. ([#6447](https://github.com/antrea-io/antrea/pull/6447), [@hongliangl])
+  - Add documentation for the new `BGPPolicy` feature. ([#6524](https://github.com/antrea-io/antrea/pull/6524), [@hongliangl])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/bgp-policy.md) for more information about this feature.
+- Add a new feature `NodeLatencyMonitor` to allow users to do east/west connectivity monitoring and provide the latency query API.
```

Comment:
to allow" -> that allows

And then change "provide" to "provides"

the latency query API -> a latency query API